conf/config-default.yaml

```diff
@@ -348,3 +348,12 @@ plugin_attr:
     report_ttl: 3600     # live time for server info in etcd (unit: second)
   dubbo-proxy:
     upstream_multiplex_count: 32
+  request-id:
+    snowflake:
+      enable: false
+      snowflake_epoc: 1609459200000   # the starting timestamp is expressed in milliseconds
+      data_machine_bits: 12           # data machine bit, maximum 31, because Lua cannot do bit operations greater than 31
```

Comment:
should check `data_machine_bits` + `sequence_bits` = 22 always? may the test case need to cover it.

Reply:
`data_machine_bits` and `sequence_bits` are not fixed and can be configured according to different requirements